modules/nixos/common/security.nix

```diff
@@ -1,7 +1,16 @@
+{ inputs, pkgs, ... }:
 {
   # Make sure that the firewall is enabled, even if it's the default.
   networking.firewall.enable = true;
 
+  # allow to access emergency shell with a password
+  boot.initrd.systemd.emergencyAccess = "$6$he2fblfl/H7I.kvz$WbSCMXu8ztmqfj5jG4czqvu/rkMHxufxqHgy1urzXFSN.jZB4QiW5lOjR08vk8pZTyim3TT1wFkMaNE9zZ3sc1";
+
+  boot.initrd.network.ssh = {
+    enable = true;
+    authorizedKeyFiles = pkgs.lib.filesystem.listFilesRecursive "${toString inputs.self}/users/keys";
```

Comment:
Since we use systemd in the initrd, I believe we need to also configure networkd here:

```
    boot.initrd.systemd.network.networks."10-uplink" = config.systemd.network.networks."10-uplink";
```

Reply:
I forgot that we'll need to set host keys for initrd as well.

Reply:
I'll merge emergency access for now and I'll do ssh later. https://github.com/nix-community/infra/pull/1401

Reply:
>  I believe we need to also configure networkd here

Looks like this is configured by default?